.github/workflows/test-dapp-encouragement.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     - name: yarn link
       run: |
         yarn link-cli ~/bin/agoric
-        echo "::add-path::/home/runner/bin"
+        echo "/home/runner/bin" >> $GITHUB_PATH
```

Comment:
I don't know how this works, but I trust you have verified it.

Reply:
Yeah, this is the recommended way to set it up, and the tests pass https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path